src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,16 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
```

Comment:
![image](https://github.com/mate-academy/hibernate-relations-hw/assets/20326858/e1403eed-4959-4240-af8e-80f2f4a52831)

Reply:
not fixed